.prettierignore

```diff
@@ -4,3 +4,10 @@ coverage
 node_modules
 API.md
 CHANGELOG.md
+
+packages/amplify-cli-core/src/__tests__/testFiles/packageManager-npm/package-lock.json
+packages/amplify-cli-core/src/__tests__/testFiles/testProject-initialize-json-error/amplify/cli.json
+packages/amplify-e2e-core/dist/index.html
+packages/amplify-e2e-tests/amplify-e2e-reports
+packages/amplify-nodejs-function-runtime-provider/src/__tests__/utils/handlerWithSyntaxError.js
+packages/amplify-velocity-template/src/parse/velocity.yy
```

Comment:
these files either:
- have intentional syntax errors
- are minified html (index.html in e2e core)
- scary to touch (velocity.yy)